tap_zoho_crm/streams/abstracts.py

```diff
@@ -200,7 +269,11 @@ def sync(
                     record, self.schema, self.metadata
                 )
 
-                record_bookmark = transformed_record[self.replication_keys[0]]
+                record_bookmark = transformed_record.get(self.replication_keys[0]) \
+                    if self.replication_keys else None
+                if record_bookmark is None:
+                    record_bookmark = bookmark_date
```

Comment:
I think below suggestion does the same thing with clean and readable implementation.

```suggestion
                if self.replication_keys:
                    record_bookmark = transformed_record.get(self.replication_keys[0])
                else:
                    record_bookmark = bookmark_date
```